### ISSUE TITLE
compilation fails in commerce 2105.7

### ISSUE THREAD

**Issue by jseidl-nc** [sap-commerce-tools/sanecleanup#3]
Hi Markus,
With commerce 2105.7 compilation of sanecleanup extension fails because CMSVersionModel._CMSVERSIONGCPROCESS2CMSVERSION doesnt exist anymore (relation CMSVersionGCProcess2CMSVersion does not exist anymore in cms2-items.xml). 

```
ERROR in /hybris/bin/custom/xyz/sanecleanup/src/mpern/sap/cleanup/cms2/CMSVersionGCPerformable.java (at line 152)
        ComposedTypeModel relationType = typeService.getComposedTypeForCode(CMSVersionModel._CMSVERSIONGCPROCESS2CMSVERSION);
                                                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_CMSVERSIONGCPROCESS2CMSVERSION cannot be resolved or is not a field
----------
1 problem (1 error)
```

Best Regards,
Joachim

**Comment by mpern** [comment]
Servus!

Thank you for the report, I totally forgot to adapt sanecleanup after the refactored GC version cleanup was released.

As a quick fix, you can just remove the whole job + config (but I don't have to tell you this 😉 )

Proper fix will come in the next few days.

**Comment by jseidl-nc** [comment]
Hi Markus,
Thanks, I think we'll wait for the proper fix :) 

Best Regards,
Joachim

**Comment by mpern** [comment]
Fixed in [v3.4.0]

[v3.4.0]:https://github.com/sap-commerce-tools/sanecleanup/releases/tag/v3.4.0
